script/Arcueid/Day 4/04_00_ARC04_6_2DASH.txt

```diff
@@ -289,7 +289,7 @@ This woman seems to be more upset by the fact that I was sleeping peacefully tha
 [sha:2f8c894ccf8ff547c50a62b91bb168cbaed41a77]{
 -- Page 466, Offset 6854.
 -- あんまりにも無防備だから、吸血種として威厳がないのかなって本気で不安になったんだから」
-You were so defenseless that I was seriously starting to lose faith in my majesty as a bloodsucker."
+You were so defenseless that I was seriously starting to believe I don't command much of a bloodsucker presence."
```

Comment:
Hmmmm, don't like the suggestion either.

The point of this change was that it was kind of a mistranslation. She's saying that she doesn't appear "bloodsuckery" enough that he'd be scared of her, essentially. 'lost my edge' is used in other contexts.

"You were so defenseless that I was seriously starting to worry I don't appear much of a bloodsucker at all." perhaps

Reply:
https://thesaurus.weblio.jp/content/%E5%A8%81%E5%8E%B3%E3%81%AE%E3%81%AA%E3%81%84

See: [オーラがない][オーラのない] for the definition applicable here.

Reply:
Yeah, I grok the intent in JP, just struggling to encode it nicely. I agree that lost her edge is more like she isn't _good_ at it anymore. Losing faith in my ability to instill a sense of awe? 
I also do think that "You were so defenseless" might do with being tweaked to something more active (you _let your guard down around me_ or along those lines), makes it a bit clearer that he's not really bothered by her rather than just that he has no means to defend against her